apps/condo/domains/banking/schema/BankCategory.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Generated by `createschema banking.BankCategory 'name:Text;'`
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File, Url } = require('@keystonejs/fields')
+const { Json, LocalizedText } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@condo/keystone/plugins')
+const access = require('@condo/domains/banking/access/BankCategory')
+
+
+const BankCategory = new GQLListSchema('BankCategory', {
```

Comment:
Why entire schema with `dv`, `sender`, `historical` and so for single field? May be `type: SELECT` is go-to solution then?